<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>payroll_generator</class>
 <widget class="QDialog" name="payroll_generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>320</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Quikmed Payroll Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/Quikmed icon/quikmed-logo.ico</normaloff>:/Quikmed icon/quikmed-logo.ico</iconset>
  </property>
  <widget class="QLineEdit" name="file_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="browse_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Summary File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Generate Payroll</string>
   </property>
  </widget>
  <widget class="QLabel" name="qm_logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icon.qrc">:/Quikmed icon/quikmed-logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Payroll Generator</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
